src/components/CartCheckout/CartCheckout.module.scss

```diff
@@ -0,0 +1,94 @@
+@import '@styles/variables';
+@import '@styles/breakpoints';
+
+.cart_page_title {
+  font-weight: 800;
+  font-size: 32px;
```

Comment:
convert to rem. default is 16px so you can connvert here to 2rem. same down below